components/event/EventCard.js

```diff
@@ -76,6 +76,14 @@ export default function EventCard({ event, username }) {
               <ReactMarkdown className="text-sm text-gray-500 py-1 flex-wrap">
                 {event.description}
               </ReactMarkdown>
+              <p className="text-sm text-gray-800 py-1 flex-wrap">
+                {event.isVirtual === true && event.isInPerson === false ? (`Remote`) : 
+                event.hasOwnProperty('location') ? (
+                  `${event.location.road!== undefined ? event.location.road + ", " : ""} 
```

Comment:
you can remove `!==undefined`

Reply:
Accordingly removed that. I tried and `!==undefined` worked. I didn't know that, I have updated that too. Thanks.

Reply:
there always something new i learn while reviewing other people's code :)